modules/postData.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable import/prefer-default-export */
+export const postData = async () => {
+  const name = document.getElementById('name');
+  const score = document.getElementById('score');
+  const res = await fetch(
+    'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/AJs6oseRhAwLgsTmDlVP/scores',
+    {
+      method: 'POST',
+      body: JSON.stringify({
+        user: name.value,
+        score: score.value,
+      }),
+      headers: { 'Content-Type': 'application/json' },
+    },
+  );
+  const data = await res.text();
+  return data;
```

Comment:
- [ ] **[OPTIONAL]** For a better experience I think it would be a good idea to clear the inputs after posting the data.